src/extension/features/accounts/transaction-grid-features/running-balance/index.js

```diff
@@ -124,48 +120,49 @@ function attachAnyItemChangedListener(accountId, transactionViewModel) {
 }
 
 function calculateRunningBalance(accountId) {
-  const accountsController = controllerLookup('accounts');
-  const transactionViewModel = accountsController.get('transactionViewModel');
+  return ynab.YNABSharedLib.defaultInstance.getBudgetViewModel_AccountTransactionsViewModel(accountId).then((transactionViewModel) => {
+    const accountsController = controllerLookup('accounts');
 
-  if (!transactionViewModel.__ynabToolKitAnyItemChangedListener) {
-    attachAnyItemChangedListener(accountId, transactionViewModel);
-  }
+    if (!transactionViewModel.__ynabToolKitAnyItemChangedListener) {
+      attachAnyItemChangedListener(accountId, transactionViewModel);
+    }
 
-  const transactions = transactionViewModel.get('visibleTransactionDisplayItems');
-  const sorted = transactions.slice().sort((a, b) => {
-    var propA = a.get('date');
-    var propB = b.get('date');
+    const transactions = transactionViewModel.get('visibleTransactionDisplayItems');
```

Comment:
Do you think this is the place where adjustments could be made regarding my comments in issue #1028 about multiple transactions on the same day for the same amount?